DataFormats/EcalDigi/src/classes_def.xml

```diff
@@ -11,21 +11,30 @@
    <class name="EcalDataFrame" ClassVersion="10">
     <version ClassVersion="10" checksum="259364627"/>
    </class>
+   <class name="EcalDataFrame_Ph2" ClassVersion="-1">
+    <version ClassVersion="-1" checksum="4122679522"/>
+   </class>
    <class name="ESDataFrame" ClassVersion="10">
     <version ClassVersion="10" checksum="2345603108"/>
    </class>
    <class name="EcalTriggerPrimitiveSample" ClassVersion="10">
     <version ClassVersion="10" checksum="251958438"/>
    </class>
-   <class name="EcalEBTriggerPrimitiveSample" ClassVersion="13">
-    <version ClassVersion="13" checksum="4171161153"/>
+   <class name="EcalEBTriggerPrimitiveSample" ClassVersion="14">
```

Comment:
Why did you change the Classversion (the `checksum` is the same)

Reply:
@perrotta  I think I had already replied to some comment about class versions. I do what scram tells me to do. Is this SO relevant ?

Reply:
Yes, this is relevant. The class version must stay unchanged if the checksum doesn't change.

Reply:
Done